lib/autobuild/import/archive.rb

```diff
@@ -183,6 +183,9 @@ def update_cache(package)
                         if retries = self.retries
                             additional_options << "--tries" << retries
                         end
+			if Autobuild.bsd?
+                            additional_options << "--no-check-certificate"
```

Comment:
On 23.03.2015 15:53, Sylvain Joyeux wrote:
>
> In lib/autobuild/import/archive.rb 
> <https://github.com/rock-core/autobuild/pull/19#discussion_r26943188>:
>
> > @@ -183,6 +183,9 @@ def update_cache(package)
> >                          if retries = self.retries
> >                              additional_options << "--tries" << retries
> >                          end
> > +			if Autobuild.bsd?
> > +                            additional_options << "--no-check-certificate"
>
> ???? More info on that ? I definitely don't want to sneak something 
> that critical like this !
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/rock-core/autobuild/pull/19/files#r26943188>.
>
It seems BSD has less certificates availble, Even with correct 
time-settings i got the error while downloading sisl



-- 
  Dipl.-Inf. Matthias Goldhoorn
  Space and Underwater Robotic

  Universität Bremen
  FB 3 - Mathematik und Informatik
  AG Robotik
  Robert-Hooke-Straße 1
  28359 Bremen, Germany
  
  Zentrale: +49 421 178 45-6611
  
  Besuchsadresse der Nebengeschäftstelle:
  Robert-Hooke-Straße 5
  28359 Bremen, Germany
  
  Tel.:    +49 421 178 45-4193
  Empfang: +49 421 178 45-6600
  Fax:     +49 421 178 45-4150
  E-Mail:  matthias.goldhoorn@informatik.uni-bremen.de

  Weitere Informationen: http://www.informatik.uni-bremen.de/robotik